frontend/src/ui/Blocks/Video.tsx

```diff
@@ -1,9 +1,40 @@
-import { BlockContainer } from ".";
-import { Player, PlayerProps } from "../player";
+import { graphql, useFragment } from "react-relay";
 
+import { Track, Player } from "../player";
+import { VideoBlockData$key } from "./__generated__/VideoBlockData.graphql";
+import { Title, BlockContainer } from ".";
 
-export const VideoBlock: React.FC<PlayerProps> = props => (
-    <BlockContainer>
-        <Player {...props} />
-    </BlockContainer>
-);
+
+type Props = {
+    title?: string;
+    fragRef: VideoBlockData$key;
+};
+
+export const VideoBlock: React.FC<Props> = ({ title, fragRef }) => {
+    const { event } = useFragment(graphql`
+        fragment VideoBlockData on VideoBlock {
+            event {
+                title
+                duration
+                tracks {
+                    uri
+                    flavor
+                    mimetype
+                    resolution
+                }
```

Comment:
One line?

Reply:
Makes it look a bit slanted to me but I don't really care. 🤷‍♀️